firefox/firefox.dockerfile

```diff
@@ -2,35 +2,9 @@ FROM janx/ubuntu-dev
 MAINTAINER Jan Keromnes "janx@linux.com"
 
 # Install Firefox build dependencies.
-# Packages are from https://dxr.mozilla.org/mozilla-central/source/python/mozboot/mozboot/debian.py
-RUN sudo apt-get update -q \
- && sudo apt-get upgrade -qy \
- && sudo apt-get install -qy \
-  autoconf2.13 \
-  build-essential \
-  ccache \
-  python-dev \
-  python-pip \
-  python-setuptools \
-  unzip \
-  uuid \
-  zip \
-  libasound2-dev \
-  libcurl4-openssl-dev \
-  libdbus-1-dev \
-  libdbus-glib-1-dev \
-  libgconf2-dev \
-  libgtk-3-dev \
-  libgtk2.0-dev \
-  libiw-dev \
-  libnotify-dev \
-  libpulse-dev \
-  libx11-xcb-dev \
-  libxt-dev \
-  mesa-common-dev \
-  python-dbus \
-  xvfb \
-  yasm
+# One-line setup command from:
+# https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Linux_Prerequisites#Most_Distros_-_One_Line_Bootstrap_Command
+RUN wget -O bootstrap.py https://hg.mozilla.org/mozilla-central/raw-file/default/python/mozboot/bin/bootstrap.py && python bootstrap.py
```

Comment:
Nit: This leaves a `bootstrap.py` file whever this command is run. Could you please explicitly download it into `/tmp`, and delete it after it's successful?

Nit: Please separate commands over multiple lines, like this:

```bash
RUN wget -O /tmp/bootstrap.py https://... \
 && python /tmp/bootstrap.py \
 && rm -f /tmp/bootstrap.py
```